### PR TITLE
test: Add more tests for `light_utils` main module

### PR DESCRIPTION
**Pull request by vadorovsky**
Achieve full coverage there.

### CODE REVIEW

File: utils/src/lib.rs [hunk 1]
```diff
@@ -122,9 +135,37 @@ mod tests {
...
-                is_smaller_than_bn254_field_size_be(&hashed_value).unwrap(),
+                is_smaller_than_bn254_field_size_be(&hashed_value),
                 "Hashed value should be within BN254 field size"
             );
         }
+
+        let max_input = [u8::MAX; 32];
+        let (hashed_value, bump) = hash_to_bn254_field_size_be(max_input.as_slice())
+            .expect("Failed to find a hash within BN254 field size");
+        assert_eq!(bump, 255, "Bump seed should be 0");
```
Comment (ananas-block) [inline utils/src/lib.rs:146]: The assert error msg confused me, shouldn't it read as follows?
```suggestion
        assert_eq!(bump, 255, "Bump seed should be 255");
```

File: utils/src/lib.rs [hunk 2]
```diff
@@ -109,11 +105,28 @@ pub fn rustfmt(code: String) -> Result<Vec<u8>, anyhow::Error> {
...
+        assert!(!is_smaller_than_bn254_field_size_be(&modulus_bytes));
+
+        let bigint = modulus.clone() - 1.to_biguint().unwrap();
+        let bigint_bytes: [u8; 32] = bigint_to_be_bytes_array(&bigint).unwrap();
+        assert!(is_smaller_than_bn254_field_size_be(&bigint_bytes));
+
+        let bigint = modulus + 1.to_biguint().unwrap();
+        let bigint_bytes: [u8; 32] = bigint_to_be_bytes_array(&bigint).unwrap();
+        assert!(!is_smaller_than_bn254_field_size_be(&bigint_bytes));
+    }
```
Comment (ananas-block) [inline utils/src/lib.rs:128]: nice!

**Review by ananas-block — COMMENTED** [review]
Thanks, just one nitpick in the assert error.

File: utils/src/lib.rs (hunk 1 above)
Comment (vadorovsky) [reply utils/src/lib.rs:146]: You're right

Comment (vadorovsky) [reply utils/src/lib.rs]: done

**Review by ananas-block — APPROVED** [review]

**Pull request merged**